src/commands/skullboard.py

```diff
@@ -2,18 +2,320 @@
 import requests
 import re
 
-from discord import Embed, Client
+from discord import Embed, Client, Interaction, AllowedMentions
 from dotenv import load_dotenv
+from datetime import datetime, timezone, timedelta
 
 from constants.colours import LIGHT_GREY
 
+import datetime as dt
+
+from discord import app_commands, Interaction
+
+from commands import database as DB
+
+
+# Adelaide timezone (UTC+9:30)
+tz = timezone(timedelta(hours=9.5))
+
+
+# Gets current day since 1970
+def get_current_day():
+    # Converts to Adelaide time
+    now = datetime.now(tz)
+    epoch = datetime(1970, 1, 1, tzinfo=tz)
+    days_since_epoch = (now - epoch).days
+    return days_since_epoch
+
+
+# Gets current day of timestamp since 1970
+def get_day_from_timestamp(timestamp: datetime):
+    # Converts to Adelaide time
+    epoch = datetime(1970, 1, 1, tzinfo=tz)
+    days_since_epoch = (timestamp - epoch).days
+    return days_since_epoch
```

Comment:
Do you need these utilities? You can probably just extract the day from the message's timestamp.

Reply:
In that case make a new folder `utils` and move these functions to there.